### PR TITLE
Contract auto refresh

### PR DESCRIPTION
**Pull request by Khalidxx**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            |  
| Patch: Bug Fix?          | Kinda (backend's bug)
| Major: Breaking Change?  | No
| Minor: New Feature?      | Yes
| Tests Added + Pass?      | Yes
| Does not reduce coverage?| Yes
| Any Dependency Changes?  |

Refreshes every contract in the trade table 3 seconds after it's sold

### CODE REVIEW

File: src/botPage/view/TradeInfoPanel/TradeTable.js [hunk 1]
```diff
@@ -19,10 +20,18 @@ const getProfit = ({ sell_price: sellPrice, buy_price: buyPrice, currency }) =>
...
+    return `${buyDate.toISOString().split('T')[0]} ${buyDate.toTimeString().slice(0, 8)} ${
+        buyDate.toTimeString().split(' ')[1]
+    }`;
+};
+
 const minHeight = 290;
 const rowHeight = 25;
 
 const ProfitColor = ({ value }) => <div style={value > 0 ? style.green : style.red}>{value}</div>;
+const StatusColor = ({ value }) => <div style={value === 'Settled' ? style.green : style.red}>{value}</div>;
```
Comment (aaron-binary) [inline src/botPage/view/TradeInfoPanel/TradeTable.js]: Make sure to run the "_Settled_" string through the `translate` function (`src/common/i18n.js`).

File: src/botPage/view/TradeInfoPanel/TradeTable.js [hunk 2]
```diff
@@ -68,16 +80,14 @@ export default class TradeTable extends Component {
...
-            }`;
+            const timestamp = getTimestamp(info.date_start);
             const tradeObj = { reference: info.transaction_ids.buy, ...info, timestamp };
             const { accountID } = tradeObj;
 
             const trade = {
                 ...tradeObj,
-                profit: getProfit(tradeObj),
+                profit         : getProfit(tradeObj),
+                contract_status: 'Pending',
```
Comment (aaron-binary) [inline src/botPage/view/TradeInfoPanel/TradeTable.js]: Same here, make sure to run the "_Pending_" string through the `translate` function.

File: src/botPage/view/TradeInfoPanel/TradeTable.js [hunk 3]
```diff
@@ -93,6 +103,45 @@ export default class TradeTable extends Component {
...
+
+            if (trade.is_expired && trade.is_sold && !trade.exit_tick) trade.exit_tick = '-';
+
+            const { id } = this.state[accountID];
+            const rows = this.state[accountID].rows.slice();
+            const updatedRows = rows.map(row => {
+                const { reference } = row;
+                if (reference === trade.reference) {
+                    return {
+                        contract_status: 'Settled',
```
Comment (aaron-binary) [inline src/botPage/view/TradeInfoPanel/TradeTable.js]: Same here, make sure to run through `translate`.

File: src/botPage/view/style.js [hunk 4]
```diff
@@ -73,9 +73,9 @@ export const bottomWarning = {
 
 export const bottomWarningLink = { textDecoration: 'underline' };
 
-export const green = { color: 'green' };
+export const green = { color: 'green', float: 'left' };
```
Comment (aaron-binary) [inline src/botPage/view/style.js]: By adding this `float` the `totalWins` and `totalLosses` cells in the account overview at the bottom of the table are incorrectly aligned.

File: src/botPage/view/TradeInfoPanel/TradeTable.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-debugger */
```
Comment (aaron-binary) [inline src/botPage/view/TradeInfoPanel/TradeTable.js]: You can remove this eslint exception.

File: src/botPage/view/TradeInfoPanel/TradeTable.js (hunk 1 above)
Comment (Khalidxx) [reply src/botPage/view/TradeInfoPanel/TradeTable.js]: Aye, capt'n!

File: src/botPage/view/style.js (hunk 4 above)
Comment (Khalidxx) [reply src/botPage/view/style.js]: Okay, I'll make separate style variables for each

**Pull request merged**